src/test/scala/com/cognite/sdk/scala/common/WritableBehaviors.scala

```diff
@@ -167,11 +167,11 @@ trait WritableBehaviors extends Matchers { this: FlatSpec =>
   )(implicit t: Transformer[R, W], t2: Transformer[R, U]): Unit =
     it should "allow updates using the read class" in {
       // create items
-      val createdItems = updatable.createFromRead(readExamples)
-      assert(createdItems.size == readExamples.size)
-      createdItems.map(_.id) should not contain 0
+      val createdIds = readExamples.map(e => updatable.createOneFromRead(e).id)
```

Comment:
> This makes several API calls instead of one

I know, but `files` endpoint don't seem to support multiple create (the `Create` trait), so I have changed the test code to insert them one by one. Is it a big deal when it's only in test code (used with 2 or 3 items)?

> If you want to DRY up the .map(_.id), do updatable.createFromRead(readExamples).map(_.id)

Not really, I wanted it to support files endpoint